app/controllers/registrations_controller.rb

```diff
@@ -327,25 +329,27 @@ def stripe_webhook
     # Create a default audit that marks the event as "unhandled".
     audit_event = StripeWebhookEvent.create_from_api(event)
     audit_remote_timestamp = audit_event.created_at_remote
+    connected_account = ConnectedStripeAccount.find_by(account_id: audit_event.account_id)
+
+    if connected_account.nil?
+      logger.error "Stripe webhook reported event for account '#{audit_event.account_id}' but we are not connected to that account."
+      return head :not_found
+    end
 
     stripe_intent = event.data.object # contains a polymorphic type that depends on the event
```

Comment:
Rename to something like `stripe_event_payload`

Reply:
Or `event_data` or whatever, I am open to whatever you think works best. Just no specific vocabulary like `intent`

Reply:
Fixed